### PR TITLE
Add App Translation Strings in Settings

### PR DESCRIPTION
**Pull request by azrikahar**
## Changes

- Allow adding custom translation strings in settings for app-wide translations
- Adds `input-traslated-strings` input interface for choosing custom translation strings

## Preview

https://user-images.githubusercontent.com/42867097/158379099-6f29583c-2046-48b2-826f-c748939c8cd7.mp4

---

Closes directus/organization#76

### CODE REVIEW

**Comment by danieljwestman** [comment]
This will be a huge improvement for us with multi lang. Great work! Seems ready to be merged? 🤓

File: app/src/hydrate.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import {
 	useUserStore,
 	useNotificationsStore,
 } from '@/stores';
+import { useTranslationStrings } from '@/modules/settings/composables/use-translation-strings';
```
Comment (rijkvanzanten) [inline app/src/hydrate.ts:19]: Seeing that this composable is used outside of the module, lets move this composable to the root composables folders

File: app/src/hydrate.ts [hunk 2]
```diff
@@ -50,6 +51,7 @@ export async function hydrate(): Promise<void> {
 	const appStore = useAppStore();
 	const userStore = useUserStore();
 	const permissionsStore = usePermissionsStore();
+	const { refresh: hydrateTranslationStrings } = useTranslationStrings();
```
Comment (rijkvanzanten) [inline app/src/hydrate.ts:54]: Maybe we can just rename the exported value from `useTranslationStrings`. Makes it easier to find and replace later if needed

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/hydrate.ts (hunk 2 above)
Comment (rijkvanzanten) [reply app/src/hydrate.ts:54]: Nah, forget it. It's cleaner to leave it as refresh in the context of that composable 👍🏻

**Review by rijkvanzanten — APPROVED** [review]
LGTM!

@azrikahar Could you do another pass on the display/interface options to see if I missed any options that should use the new interface (and maybe see if we can update `list` to also use it?) Feel free to merge this afterwards 🙂

**Comment by azrikahar** [comment]
@rijkvanzanten here are the following changes, will leave the merging to you once you have reviewed them 👍 (feel free to ignore some/all of the changes that are correct)

- Noticed the `Translation Strings` field is showing up at the end of the Project Settings page, opted to hide it here. However I can also see the value of being able to copy/paste the raw value, but then again maybe this is another potential enhancement that should (somehow) be doable in the Translation Strings page instead.

  Preview of Translation Strings if not hidden:
  
  chrome_YV1oPqmM3o
  
- Noticed the interface is showing up in string fields, but since it was moved to system interfaces as you mentioned, added `system: true` to hide it. 

- Used this new interface for field detail notes:

  chrome_bq6f8Is4dk
 
- For the list or repeater, added the new interface for "Create New" label and "note" of each field:

  "Create New" label:
  
   chrome_Fs8Fa0uY8N

  Notes of each field:
  
   chrome_kYMJP578QC

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**